configure-repo fails when running on a fresh repo

The configure-repo command fails when run against a fresh repository:

```
% github-keeper configure-repo metabase-driver

metabase-driver 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13d3084]

goroutine 1 [running]:
github.com/exasol/github-keeper/cmd.BranchProtectionVerifier.CheckIfBranchProtectionIsApplied({{0x7ffeefbffb4d, 0xc000128008}, 0xc0001a8000}, 0x5)
	/Users/chp/go/pkg/mod/github.com/exasol/github-keeper@v0.0.0-20211118101907-26b675c2312d/cmd/branchProtection.go:52 +0xa4
github.com/exasol/github-keeper/cmd.glob..func1(0x189f220, {0xc0001110f0, 0x1, 0x1})
	/Users/chp/go/pkg/mod/github.com/exasol/github-keeper@v0.0.0-20211118101907-26b675c2312d/cmd/configureRepo.go:28 +0x15e
github.com/spf13/cobra.(*Command).execute(0x189f220, {0xc0001110d0, 0x1, 0x1})
	/Users/chp/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x189f720)
	/Users/chp/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/chp/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/exasol/github-keeper/cmd.Execute()
	/Users/chp/go/pkg/mod/github.com/exasol/github-keeper@v0.0.0-20211118101907-26b675c2312d/cmd/root.go:16 +0x25
main.main()
	/Users/chp/go/pkg/mod/github.com/exasol/github-keeper@v0.0.0-20211118101907-26b675c2312d/main.go:6 +0x17
```

--- a/cmd/branchProtection.go
+++ b/cmd/branchProtection.go
@@ -49,9 +49,8 @@ func (verifier BranchProtectionVerifier) CheckIfBranchProtectionIsApplied(fix bo
 	problemHandler := verifier.getProblemHandler(fix)
 	repo := verifier.getRepo()
 	branch := *repo.DefaultBranch
-	language := *repo.Language
 	existingProtection, resp, _ := verifier.client.Repositories.GetBranchProtection(context.Background(), "exasol", verifier.repoName, branch)
-	protectionRequest := verifier.createProtectionRequest(verifier.isSonarRequired(language))
+	protectionRequest := verifier.createProtectionRequest(verifier.isSonarRequired(repo.Language))
 	if resp.StatusCode == 404 {
 		problemHandler.createBranchProtection(verifier.repoName, branch, &protectionRequest)
 	} else {
@@ -66,8 +65,8 @@ func (verifier BranchProtectionVerifier) CheckIfBranchProtectionIsApplied(fix bo
 	}
 }
 
-func (verifier BranchProtectionVerifier) isSonarRequired(language string) bool {
-	return language == "Scala" || language == "Java" || language == "Go"
+func (verifier BranchProtectionVerifier) isSonarRequired(language *string) bool {
+	return language != nil && (*language == "Scala" || *language == "Java" || *language == "Go")
 }
 
 func (verifier BranchProtectionVerifier) getRepo() *github.Repository {

--- a/cmd/sliceComparator_test.go
+++ b/cmd/sliceComparator_test.go
@@ -21,6 +21,10 @@ func (suite SliceComparatorSuite) TestEqualWithDifferentOrder() {
 	suite.Assert().True(stringSlicesEqualIgnoringOrder([]string{"a", "b"}, []string{"b", "a"}))
 }
 
+func (suite SliceComparatorSuite) TestEqualWithDifferentLength() {
+	suite.Assert().False(stringSlicesEqualIgnoringOrder([]string{"a", "b"}, []string{"a", "b", "c"}))
+}
+
 func (suite SliceComparatorSuite) TestDifferent() {
 	suite.Assert().False(stringSlicesEqualIgnoringOrder([]string{"a", "b"}, []string{"a", "c"}))
 }

--- a/doc/changes/changes_0.1.0.md
+++ b/doc/changes/changes_0.1.0.md
@@ -21,6 +21,7 @@ Code name: Improved GitHub Actions
 
 * #16: Fixed unify-labels for deprecated labels where the replacement exists
 * #19: Fixed create-branch-protection for duplicated check names
+* #29: Fixed configure-repo for repositories with no detected language
 
 ## Refactoring
 

